### PR TITLE
Rename transactionItem capture amount to charge amount

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it changes the name of input and model fields from captured_amount to charged_amount. The `charge` is more meaningful and not as much related to credit card payment as capture. 
This is not a breaking change, as an interface for transactions was not yet released. 

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-7007 Rename TransactionItem.capturedAmount field into chargedAmount](https://app.clickup.com/t/2549495/SALEOR-7007)

File: saleor/payment/migrations/0036_auto_20220518_0732.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+def update_available_actions(apps, schema_editor):
+    TransactionItem = apps.get_model("payment", "TransactionItem")
+    transactions = TransactionItem.objects.filter(
+        available_actions__contains=["capture"]
+    )
+    transaction_to_update = []
```
Comment (korycins) [inline saleor/payment/migrations/0036_auto_20220518_0732.py:12]: It is simple `for` as the API for creating transactions hasn't been released yet, but everyone who would use the `main` branch could have an issue without this.

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/order/mutations/order_confirm.py [hunk 2]
```diff
@@ -66,10 +66,10 @@ def perform_mutation(cls, root, info, **data):
                 # We use the last transaction as we don't have a possibility to
                 # provide way of handling multiple transaction here
                 payment_transaction = payment_transactions[-1]
-                request_capture_action(
+                request_charge_action(
                     transaction=payment_transaction,
                     manager=info.context.plugins,
-                    capture_value=payment_transaction.captured_value,
+                    charge_value=payment_transaction.authorized_value,
```
Comment (SzymJ) [inline saleor/graphql/order/mutations/order_confirm.py:72]: why `authorized_value` ?

Comment (korycins) [reply saleor/graphql/order/mutations/order_confirm.py:72]: This is a logic that calls a charge request action. Previously we were calling capture requests with the amount that is already captured.

**Review by SzymJ — APPROVED** [review]

**Pull request merged**